src/features/goal/list.tsx

```diff
@@ -0,0 +1,32 @@
+import { Eraser } from "phosphor-react";
+import { trpc } from "../../utils/trpc";
+import { IconButton } from "../ui/button";
+
+type Goal = { id: string; name: string };
+
+export const GoalList = ({ goals }: { goals: Goal[] }) => {
+  if (goals.length === 0)
+    return <p className="text-xs leading-6">Your goal list is empty!</p>;
```

Comment:
We can write
"How can a man lead his life without any goal?"
😄

Reply:
Nice suggestion, I'll change it 👍

Reply:
![image](https://user-images.githubusercontent.com/47036679/205944501-72af5938-50b8-4e42-a53d-0ea76df6c854.png)